src/routes/init/index.ts

```diff
@@ -1,20 +1,23 @@
 import { Request } from 'express';
 import { DbClient, IDbClient } from '../../dal/interface';
 import { InsertWriteOpResult } from 'mongodb';
-import { data } from './output';
+import { importDbEntries } from './import';
 import { applyTracing } from '../../utils/tracer/index';
 import { CSEResponse } from '../../server/index';
+import { IClaim } from '../../models/claim/index';
 
 
 export const handler = async (req: Request, res: CSEResponse): Promise<void> => {
+    const output: IClaim[] = await importDbEntries();
     const dbArgs: IDbClient = {
         url: 'mongodb://mongo',
         port: 27017,
         dbName: 'cse'
     };
     const client: DbClient = new DbClient(dbArgs);
```

Comment:
Instead of declaring a local DbServer here, you should use the one available to each request in `res.locals`. See the `CSEResponse` interface to see what else is available to each request handler. 
```
const { dbClient } = res.locals;
```